report_generator/metrics_report_gen_config.json

```diff
@@ -0,0 +1,35 @@
+{

```

Comment:
why metrics? Is this created to collect all the metrics? There's a lot of extra parameters for GB then

Reply:
rename to sklearn_metrics